.github/test-categories.yaml

```diff
@@ -105,6 +105,7 @@ platform:
   - platform-http-engine
   - reactive-streams
   - rest-openapi
+  - threadpoolfactory-vertx
```

Comment:
Because there is no dedicated integration test module for `threadpoolfactory-vertx`. You may need to add it to the list of exclusions in `scripts/validate-github-workflows.groovy` in order to prevent the validation checks from failing.